Privacy Module is failing

## The Bug
Log:
```
00:38:24.117 	main 	INFO 	Arisa 	99 	[RESPONSE] [MCD-3701] [Privacy] Successful
00:38:24.345 	main 	ERROR 	Arisa 	108 	[UPDATE] [MCD-3701] Failed
net.rcarz.jiraclient.JiraException: Failed to transition issue MCD-3701
    at net.rcarz.jiraclient.Issue$FluentTransition.realExecute(Issue.java:484)
    at net.rcarz.jiraclient.Issue$FluentTransition.execute(Issue.java:518)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$applyFluentTransition$1$1.invokeSuspend(Functions.kt:128)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$applyFluentTransition$1$1.invoke(Functions.kt)
    at arrow.core.Either$Companion.catch(Either.kt:862)
    at arrow.core.Either$Companion.catch(Either.kt:858)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$applyFluentTransition$1.invokeSuspend(Functions.kt:127)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
    at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:271)
    at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:79)
    at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:54)
    at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
    at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:36)
    at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt.applyFluentTransition(Functions.kt:126)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt.access$applyFluentTransition(Functions.kt:1)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$applyIssueChanges$2.invoke(Functions.kt:108)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt$applyIssueChanges$2.invoke(Functions.kt)
    at arrow.syntax.function.PartialsKt$partially1$2.invoke(partials.kt:773)
    at arrow.syntax.function.PartialsKt$partially1$1.invoke(partials.kt:771)
    at io.github.mojira.arisa.modules.HelpersKt.tryRunAll(Helpers.kt:75)
    at io.github.mojira.arisa.infrastructure.jira.FunctionsKt.applyIssueChanges(Functions.kt:117)
    at io.github.mojira.arisa.ModuleExecutor.executeModule(ModuleExecutor.kt:104)
    at io.github.mojira.arisa.ModuleExecutor.execute(ModuleExecutor.kt:49)
    at io.github.mojira.arisa.ArisaMainKt.main(ArisaMain.kt:67)
    at io.github.mojira.arisa.ArisaMainKt.main(ArisaMain.kt)
Caused by: net.rcarz.jiraclient.RestException: 400 : {"errorMessages":[],"errors":{"security":"Could not find valid 'id' or 'name' in security level object."}}
    at net.rcarz.jiraclient.RestClient.request(RestClient.java:171)
    at net.rcarz.jiraclient.RestClient.request(RestClient.java:193)
    at net.rcarz.jiraclient.RestClient.request(RestClient.java:238)
    at net.rcarz.jiraclient.RestClient.post(RestClient.java:328)
    at net.rcarz.jiraclient.RestClient.post(RestClient.java:372)
    at net.rcarz.jiraclient.Issue$FluentTransition.realExecute(Issue.java:482)
     ... 26 common frames omitted 
```
## Example Ticket
[MCD-3705](https://bugs.mojang.com/browse/MCD-3705)
[MCD-3701](https://bugs.mojang.com/browse/MCD-3701)
[MC-190820](https://bugs.mojang.com/browse/MC-190820)
[REALMS-4184](https://bugs.mojang.com/browse/REALMS-4184)

## Expected behavior
Don't fail.

--- a/src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Functions.kt
+++ b/src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Functions.kt
@@ -75,8 +75,8 @@ fun resolveAs(context: Lazy<IssueUpdateContext>, resolution: String) {
 }
 
 fun updateSecurity(context: Lazy<IssueUpdateContext>, levelId: String) {
-    context.value.hasUpdates = true
-    context.value.update.field(Field.SECURITY, Field.valueById(levelId))
+    context.value.hasEdits = true
+    context.value.edit.field(Field.SECURITY, Field.valueById(levelId))
 }
 
 fun removeAffectedVersion(context: Lazy<IssueUpdateContext>, version: Version) {

